app/serializers/list_serializer.rb

```diff
@@ -0,0 +1,9 @@
+class ListSerializer < ActiveModel::Serializer
+  attributes :id, :title, :user_id, :public, :created_at
+  has_many :items
```

Comment:
yeah no relationships here either...

ur model deals w all this

and created_at is smething that is autogenerated remember when a new object is created

so ur model for list shold look like so..

```
class List < ApplicationRecord
  belongs_to :user
  has_many :items, dependent: :destroy
  validates :title, presence: true
end
```